chainstate/src/detail/tests/processing_tests.rs

```diff
@@ -147,9 +119,9 @@ fn test_orphans_chains() {
     });
 }
 
-// Create the chainstate without a genesis block.
 #[test]
-fn empty_chainstate() {
+#[should_panic(expected = "Best block ID not initialized")]
+fn test_empty_chainstate_no_genesis() {
```

Comment:
I think that the "test" prefix is needless. There is the "`#[test]` attribute right here and the output is also pretty obvious: "running tests... test <test_name>". I suppose that `chainstate_no_genesis` is enough here.